deploy-agent/deployd/__init__.py

```diff
@@ -18,5 +18,6 @@
 IS_PINTEREST = True if os.getenv("IS_PINTEREST", "false") == "true" else False
 METRIC_PORT_HEALTH = int(os.getenv('METRIC_PORT_HEALTH')) if os.getenv('METRIC_PORT_HEALTH', False) else None
 METRIC_CACHE_PATH = os.getenv('METRIC_CACHE_PATH', None)
+TELEFIG_BINARY = os.getenv('TELEFIG_BINARY', "")
```

Comment:
We don't have this env variable.

Reply:
we will need to update puppet to have this env.